packages/www/components/AssetDetails/AssetEventLogTab.tsx

```diff
@@ -30,15 +30,7 @@ const AssetEventLogTab = ({ asset }: { asset: Asset }) => {
               task.outputAssetId === asset.id || task.inputAssetId === asset.id
           )
           ?.map((task) => ({
-            type: task?.params?.import?.url
```

Comment:
@0xcadams @victorges do you know if this part of the code is still relevant? I've just removed the lines that rely on the now deleted import task

Reply:
Hmm, maybe it would still be relevant to show the info from the old import tasks, tho those are so far in the past now I bet no one will be looking at them anymore. I say we just clean this code as wel..

Reply:
WDYT of transforming this `import` based logic into `upload` tho? Should be relatively easy (tho would be a new feature in a sense, not sth we have today. totally optional)

Reply:
Yeah this is what the event tab is looking like today for a freshly uploaded asset, looks pretty buggy to me tbh (with `import` task it shows an event of 'asset uploaded' etc). Would be a nice to have to fix this, tho as I said it would not be a regression to just leave as is if you prefer, and we can later prioritize fixing this events tab.

<img width="755" alt="Screenshot 2023-09-25 at 10 51 47" src="https://github.com/livepeer/studio/assets/1613383/5d03de56-97de-46ee-b5e9-5abc265a295e">

Reply:
Ah gotcha, i'll take a look